Formula/gegl.rb

```diff
@@ -24,23 +24,33 @@ class Gegl < Formula
   depends_on "babl"
   depends_on "gettext"
   depends_on "glib"
+  depends_on "jpeg"
   depends_on "json-glib"
   depends_on "libpng"
-  depends_on "jpeg"
   depends_on "cairo" => :optional
+  depends_on "jasper" => :optional
   depends_on "librsvg" => :optional
   depends_on "lua" => :optional
   depends_on "pango" => :optional
   depends_on "sdl" => :optional
+  depends_on "suite-sparse" => :optional
```

Comment:
rather than adding new options no one has yet asked for, let's just always pass those arguments to configure

Reply:
Personally, I feel that it’s better to keep the options in because then, if somebody wants it, it’s already incorporated and they don’t have to wait/add it in themselves, and it also self-documents the arguments a bit more, but it’s up to you!  I’m about to head to bed, but I’ll tackle it first thing in the morning.  😉